content/en/docs/concepts/overview/working-with-objects/common-labels.md

```diff
@@ -54,6 +55,7 @@ metadata:
     app.kubernetes.io/component: database
     app.kubernetes.io/part-of: wordpress
     app.kubernetes.io/managed-by: helm
+    app.kubernetes.io/created-by: helm
```

Comment:
Is `helm` the correct value?

Reply:
I forgot to change it to the example above, thank you :)